src/main/groovy/com/bmuschko/gradle/docker/tasks/image/DockerSaveImage.groovy

```diff
@@ -30,8 +30,10 @@ class DockerSaveImage extends AbstractDockerRemoteApiTask {
 
     @Override
     void runRemoteCommand(Object dockerClient) {
-        if (!destFile || !destFile.isFile()) {
-            throw new GradleException("Invalid destination file: " + destFile)
+        boolean success = destFile.createNewFile()
```

Comment:
Think we might want to do something like:
```
if (!destFile.exists()) {
    if(!destFile.createNewFile()) {
        throw new GradleException("Could not create file @ ${destFile.path}")
    }
}

if (destFile.isDirectory()) {
    throw new GradleException("destFile cannot be a directory")
}
```

Reply:
Sounds good !